recording-daemon/decoder.c

```diff
@@ -378,7 +379,12 @@ void decoder_close(decoder_t *dec) {
 	if (!dec)
 		return;
 	/// XXX drain inputs and outputs
+#if LIBAVCODEC_VERSION_INT >= AV_VERSION_INT(56, 1, 0)
 	avcodec_free_context(&dec->avcctx);
+#else
+	avcodec_close(dec->avcctx);
+	av_free(dec->avcctx);
+#endif
```

Comment:
So, this is to be able to build with older versions of libav/ffmeg, which would seem to confirm using even older versions might indeed not be supported?